R/reactives.R

```diff
@@ -544,14 +553,17 @@ Observer <- R6Class(
 
     initialize = function(observerFunc, label, suspended = FALSE, priority = 0,
                           domain = getDefaultReactiveDomain(),
-                          autoDestroy = TRUE) {
+                          autoDestroy = TRUE, ..stacktraceon = TRUE) {
```

Comment:
I'm not sure I like the name `..stacktraceon` for the argument name. I see the parallel with the function name, but I'm not sure it actually helps, since "stacktraceon" is already pretty distinctive.